src/CSS/CreateComment.css

```diff
@@ -0,0 +1,95 @@
+.detailBox {
+    width: 84%;
+    border: 1px solid #bbb;
+    margin-top: 50px;
+    margin-left: 91px;
+
+}
+
+.titleBox {
+    padding:10px;
+}
+.commentBox {
+    padding:10px;
+    border-top:1px dotted #bbb;
+}
+.commentBox .form-group:first-child, .actionBox .form-group:first-child {
+    width:80%;
+}
+.commentBox .form-group:nth-child(2), .actionBox .form-group:nth-child(2) {
+    width:18%;
+}
+.actionBox .form-group * {
+    width:100%;
+}
+.taskDescription {
+    margin-top:10px 0;
+}
+.commentList {
+    padding:0;
+    list-style:none;
+    max-height:450px;
+    overflow:auto;
+}
+.commentList li {
+    margin:0;
+    margin-top:20px;
+}
+.commentList li > div {
+    display:table-cell;
+}
+.commenterImage {
+    width:30px;
+    margin-right:5px;
+    height:100%;
+    float:left;
+    width: 50px;
+}
+.commenterImage img {
+    width:100%;
+    border-radius: 50%;
+    height: 48px;
+    margin-top: 7px;
+}
+.commentText p {
+    margin-top: -6px;
+    margin-left: 8px;
+    margin-right: 25px;
+    text-align: justify;
+    padding: 15px;
+}
+.commentCover {
+    background-color: #dde1e4;
+    margin-bottom: auto;
+    border-radius: 5px;
+    width: 825px;
+    margin-left: 9px;
+}
+.sub_text {
+    color:#aaa;
+    font-family:verdana;
+    font-size:11px;
+}
+.actionBox {
+    border-top:1px dotted #bbb;
+    padding:10px;
+}
+.loading {
+    color: gray;
+}
+.CommentersName {
+    color: #0693e3;
+    text-decoration: none;
+}
+.loadMore {
+    color: black;
+}
+.loadMore:hover {
+    color: #717477;
+}
+.CommentersName:hover {
+    color: #00bcd4;
+}
+.errors {
+    color: red;
+}
```

Comment:
eof misssing here

Reply:
That file already has an EOF on it, please check it again @RutaleIvanPaul.

Reply:
All my files have EOF lines on them, please check and tell me which one in particular!